### DISCUSSION TITLE
Question about coqui.ai new voice creation and emotion

### DISCUSSION THREAD

**Discussion by OnceJune** [coqui-ai/TTS#2466] [General Q&A] [answered]
Hi, I tried the coqui.ai and find it is amazing. I have some questions about the studio:
1. Every voice including the new created ones has difference emotions, how do you achieve this?
2. How does the voice fusion work?
3. Prompt-to-voice is fantastic, is it implemented in this repo?
Thanks in advance.

**Comment by kdavis-coqui** [comment, marked as answer]
Trade secrets. Sorry.

**Reply by OnceJune** [reply]
Thank you
